xbmc/rendering/Extensions.h

```diff
@@ -0,0 +1,53 @@
+/*
+ *  Copyright (C) 2017-2018 Team Kodi
+ *  This file is part of Kodi - https://kodi.tv
+ *
+ *  SPDX-License-Identifier: GPL-2.0-or-later
+ *  See LICENSES/README.md for more information.
+ */
+
+#pragma once
+
+#include "utils/Map.h"
+
+#include <string_view>
+
+struct GLEXTENSIONS
+{
+  enum EXTENSION
```

Comment:
This email and values don't follow the normal conventions https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#83-enums